### PR TITLE
SIG: upgrade `SIG-Expr` to `SIG-Exec` and update the member list

### PR DESCRIPTION
**Pull request by qw4990**
Upgrade `SIG-Expr` to `SIG-Exec` and update the member list.

### CODE REVIEW

**Comment by gingerkidney** [comment]
should we just delete the old files? @zz-jason

File: special-interest-groups/sig-exec/roles-and-organization-management.md [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
 
 ### Promotion
 
 #### From `Contributor` to `Active Contributor`
 
 Requirements:
 
-* Have at least 8 PRs merged in the expression package within one year.
-* Nominated by at least 2 Expr SIG Reviewers, Committer or Maintainer.
+* Have at least 8 PRs merged in the expression package or 4 PRs merged in the executor package within one year.
```
Comment (zz-jason) [inline special-interest-groups/sig-exec/roles-and-organization-management.md]: @qw4990 should we obey the common definition about `active contributor`?
@dcalvin please take a look.

**Comment by zz-jason** [comment]
> should we just delete the old files? @zz-jason

@gingerkidney Why removing the old files? I think we can reuse them by renaming `expr` to `exec` just like what this PR does.

File: special-interest-groups/sig-exec/roles-and-organization-management.md (hunk 1 above)
Comment (dcalvin) [reply special-interest-groups/sig-exec/roles-and-organization-management.md]: Yeah here we only list the SIG specific requirements. If there is nothing specific here, we should just refer to the common definition.

**Comment by gingerkidney** [comment]
LGTM

File: special-interest-groups/README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ SIG.
 
 ## Active Special Interest Groups
 
-* [sig-expr](./sig-expr)
+* [sig-exex](./sig-exex)
```
[the remark this answers is not in the document]
Comment (qw4990) [reply special-interest-groups/README.md]: Sorry my mistake, it has been fixed.

**Pull request merged**